src/serverMain/kotlin/com/studo/campusqr/endpoints/SeatFilter.kt

```diff
@@ -19,7 +19,7 @@ import java.util.*
 
 // Edit or create seat filter
 suspend fun AuthenticatedApplicationCall.editSeatFilter() {
```

Comment:
seat filter is a reporting feature. so i would say that we should use the viewCheckIn permission